# Image Processing
close_point:         0.1
far_point:           50.0
min_intensity:       2
max_intensity:       3000
flatness_range:      1.0
flatness_intensity:  5.0

# Input / Output
in_pointcloud_topic: /os_cloud_node/points
out_image_topic:     /os_cloud_node/images

# LiDAR Settings
projection_height:   64
projection_width:    2048

# General Settings
visualize:           False
resize_output:       False
